ballerina-shell/modules/shell-cli/src/main/java/io/ballerina/shell/cli/jline/parser/ParserStateMachine.java

```diff
@@ -160,8 +165,14 @@ private void normalState(char character) {
                 break;
             default:
         }
-        if (CONTINUING_OPERATORS.contains(character)) {
-            state = ParserState.AFTER_OPERATOR;
+        // Change the transitioning state according to the current character.
+        // If current character is an operator, the next state would be AFTER_OPERATOR.
+        if (state == ParserState.NORMAL || state == ParserState.AFTER_OPERATOR) {
```

Comment:
No it can be changed in the case statements, for example in https://github.com/ballerina-platform/ballerina-lang/blob/128a34beda6b9586c45758b1904a7ce0f2668c87/ballerina-shell/modules/shell-cli/src/main/java/io/ballerina/shell/cli/jline/parser/ParserStateMachine.java#L150-L151

Reply:
My bad, missed that.